Makefile.osl

```diff
@@ -0,0 +1,328 @@
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 1.30.0
```

Comment:
can you sync with the `Makefile`. We introduced a new `REDUCED_VERSION` in there

Why not use the `include Makefile` and redefine the needed targets only ?

Reply:
Unfortunately I noticed that the `include Makefile` works for targets but not for variables and so even if we can override the targets there is an issue with the variables.